README.md

```diff
@@ -13,8 +13,8 @@ If you haven't installed singularity, do that with [these instructions](http://s
 
 Let's now create a jupyter notebook!
 
-      sudo singularity create --size 4000 jupyter.img
-      sudo singularity bootstrap jupyter.img Singularity
+      sudo singularity image.create --size 4000 jupyter.img
+      sudo singularity build jupyter.img Singularity
 
```

Comment:
Do we want to remote the image.create command entirely? Is it needed?

Reply:
When I run the current command in Singularity 2.4 it said that the "create" command is depreciated, and image.create is recommended for this newer version of Singularity. Is there some point that I am missing?

Reply:
:)